spec/services/measurements_creator_spec.rb

```diff
@@ -0,0 +1,148 @@
+require "spec_helper"
+
+describe MeasurementsCreator do
+  describe "#self.call" do
+    before() do
+      @arrival_utc_time = Time.utc(2018, 11, 19)
```

Comment:
The `@arrival_utc_time` var is used in only one test. I'd suggest you stub directly there to improve readability and have less noise in the other tests

Reply:
It's also used on line 7

Reply:
I see but only one test is using that, correct?